src/hooks/useGetAboutTheAmazonContent.tsx

```diff
@@ -1082,6 +1095,79 @@ export function useGetAboutTheAmazonContent() {
         },
       },
     },
+    {
+      type: "regular",
```

Comment:
Will you add the data from Sanity in a different PR?

Reply:
Yes

Reply:
No need, you can add it in a different PR